simapp/go.mod

```diff
@@ -214,6 +214,8 @@ require (
 replace (
 	cosmossdk.io/api => ../api
 	cosmossdk.io/log => ../log
+	cosmossdk.io/x/feegrant => ../x/feegrant/.
+	cosmossdk.io/x/nft => ../x/nft/.
```

Comment:
Some x/modules come with their own go.mod. Until they are removed or there is a new release, these replace statements ensure that the latest version of this branch is used